src/components/molecules/ScheduleItemNG/ScheduleItemNG.tsx

```diff
@@ -183,9 +179,12 @@ export const ScheduleItemNG: React.FC<ScheduleItemNGProps> = ({
           </>
         )}
       </div>
-      {isLive && (
+      {isCurrentEventLive && (
         <div className="ScheduleItemNG__online">
-          <FontAwesomeIcon icon={solidUsers} />
+          <FontAwesomeIcon
+            className="ScheduleItemNG__online__icon"
```

Comment:
```suggestion
            className="ScheduleItemNG__online-icon"
```
to preserve BEM convention of single dunder per class name.

Reply:
thanks, fixed c035445